source/inmemory.rst

```diff
@@ -46,28 +46,52 @@ have to do one of the following:
      This is controlled using
      the :option:`--inMemoryStatisticsLogDelaySecs` option.
 
-* If you want to permanently switch to Percona Memory Engine and do not have any
-  valuable data in your database, clean out the default data directory and edit
-  the configuration file:
+* If you want to permanently switch to |inmemory| and do not have any
+  valuable data in your database, do the following:
+
+  Clean out the default data directory:
 
   .. code-block:: bash
 
      $ service mongod stop
      $ rm -rf /var/lib/mongodb/*
-     $ sed -i '/engine: .*inMemory/s/#//g' /etc/mongod.conf
-     $ service mongod start
 
-* If there is data that you want to migrate and make compatible with Percona
-  Memory Engine, use the ``mongodump`` and ``mongorestore`` utilities:
+  Edit the configuration file:
 
   .. code-block:: bash
-
-     $ mongodump --out <dumpDir>
-     $ service mongod stop
-     $ rm -rf /var/lib/mongodb/*
+		 
      $ sed -i '/engine: .*inMemory/s/#//g' /etc/mongod.conf
      $ service mongod start
-     $ mongorestore <dumpDir>
+
+  The :bash:`sed -i '/engine: .*inMemory/s/#//g' /etc/mongod.conf` command uncomments the lines
+  that contain the ``engine: .*inMemory`` pattern in the :file:`/etc/mongod.conf`.
+  
+* If there is data that you want to migrate and make compatible with |inmemory|, use the following methods:
+
+  - for replicasets, use the "rolling restart" process. 
+    Switch to the |inmemory| on the secondary node. Clean out the default data directory and edit the configuration file:
+
+    .. code-block:: bash
+
+       $ service mongod stop
+       $ rm -rf /var/lib/mongodb/*
```

Comment:
I think we should say "<the dbpath directory>". It will often be something than the default. Most users in my experience change to be on a big mount at path such as "/mongo" or "/data".